ui/tests/test-helper.js

```diff
@@ -10,24 +10,21 @@ import { setApplication } from '@ember/test-helpers';
 import { setup } from 'qunit-dom';
 import start from 'ember-exam/test-support/start';
 import './helpers/flash-message';
-import preloadAssets from 'ember-asset-loader/test-support/preload-assets';
 import { setupGlobalA11yHooks, setRunOptions } from 'ember-a11y-testing/test-support';
-import manifest from 'vault/config/asset-manifest';
 
-preloadAssets(manifest).then(() => {
```

Comment:
we removed the lazyLoading in tests and so we no longer need to preload the assets